cache_run_c_code = 'Test does not build: Traceback (most recent call last):\n  File "/home/binhn/lena/.waf-1.7.10-4f6df1d839dc35640834d81573053140/waflib/Tools/c_config.py", line 459, in run_c_code\n  File "/home/binhn/lena/.waf-1.7.10-4f6df1d839dc35640834d81573053140/waflib/Build.py", line 188, in compile\nBuildError: Build failed\n -> task in \'testprog\' failed (exit status 1): \n\t{task 4146737484L: cxx test.cpp -> test.cpp.1.o}\n[\'/usr/bin/g++\', \'-O0\', \'-ggdb\', \'-g3\', \'-Wall\', \'-Werror\', \'-DNS3_ASSERT_ENABLE\', \'-DNS3_LOG_ENABLE\', \'../test.cpp\', \'-c\', \'-o\', \'test.cpp.1.o\']\n'
